Client/Cliqz/Frontend/Resources/fr.lproj/Cliqz.strings

```diff
@@ -192,46 +192,46 @@
 "When you share your location, you also get results for locations near you (Google Maps)" = "When you share your location, you also get results for locations near you (Google Maps)";
 
 /* Start buttun title for onBorading */
-"Let's start!" = "Let's start!";
+"Let's start!" = "Commencer!";
 
 /* Toast message shown when youtube video download started */
-"The video is being downloaded." = "The video is being downloaded.";
+"The video is being downloaded." = "La vidéo a été téléchargée.";
 
 /* Toast message shown when youtube video download faild */
-"The download failed." = "The download failed.";
+"The download failed." = "Échec lors du téléchargement.";
 
 /* Toast message shown when youtube video is Successfully downloaded */
-"The download is complete. Open the Photos app to watch the video." = "The download is complete. Open the Photos app to watch the video.";
+"The download is complete. Open the Photos app to watch the video." = "Le téléchargement est fini. Ouvrir l'application «Vidéo» afin de visionner la vidéo.";
 
 /* Brading label for onBorading */
-"Faster and safer" = "Faster and safer";
+"Faster and safer" = "Plus rapide et plus sûr";
 
 /* subtitle label for onBorading */
-"SEARCH ENGINE \n IN THE BROWSER" = "SEARCH ENGINE \n IN THE BROWSER";
+"SEARCH ENGINE \n IN THE BROWSER" = "MOTEUR DE RECHERCHE \n DANS LE NAVIGATEUR";
 
 /* Anti-tracking hint title */
 "Anti-Tracking" = "Anti-Tracking";
 
 /* Anti-tracking hint description */
-"Anti-Tracking description" = "You are in control: CLIQZ Anti-Tracking protects your privacy";
+"Anti-Tracking description" = "Vous êtes au contrôle: Anti-Tracking de Cliqz protège votre vie privée";
 
 /* Cliqz search hint title */
-"Fast-search" = "Fast-Search";
+"Fast-search" = "Recherche rapide";
 
 /* Cliqz search hint description */
-"Fast-search description" = "Everything now. No detours";
+"Fast-search description" = "Tout maintenant. Sans détour";
 
 /* Detailed description on Adblocking settings */
-"AdBlocking settings clarification" = "Please note: Cliqz Browser has to download some data packages first, before the ad-blocker works efficiently. For this, the app must be connected to Wi-Fi.";
+"AdBlocking settings clarification" = "À noter: le navigateur Cliqz doit d'abord télécharger certains composants afin de mettre en marche le bloqueur de pubs. Pour cela, vous devez être connecté au Wi-Fi.";
```

Comment:
note` `:
Is this message still used, I think we now ship the resources with the app.